pkg/policy/snapshots.go

```diff
@@ -207,19 +207,21 @@ func (ce *Executor) checkTableExistence(ctx context.Context, tableName string) (
 	explainQuery := fmt.Sprintf("select pg_get_viewdef('%s'::regclass::oid) ", tableName)
 	rows, err := ce.conn.Query(ctx, explainQuery)
 	if err != nil {
+		ce.log.Error("error running explain", "tableName", tableName)
 		return "", err
 	}
+	if err := rows.Err(); err != nil {
```

Comment:
As per doc, `rows.Err` returns errors occured while reading, so moving this before the reading part (which is `Scan`) is not correct? But there's a similar thing in line 152 of the same file where the rows.Err block should be moved to after rows.Next around line 111.

Reply:
Thanks for that catch! Fixed it